azure/packages/external-controller/package.json

```diff
@@ -44,14 +44,14 @@
   "dependencies": {
     "@fluidframework/azure-client": "^1.1.0",
     "@fluidframework/common-definitions": "^0.20.1",
+    "@fluidframework/test-client-utils": "^1.0.1",
```

Comment:
This has to stay in the dev dependencies because of layer-check. See #10226.

Reply:
Okay. I'll revert the change then.